public/ACMW/css/members.css

```diff
@@ -90,10 +90,19 @@
 	color: #fff;
 	background-color: #555 !important;
 }
+footer .container{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The properties padding-top, padding-bottom, padding-left, padding-right can be replaced by padding.](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)